app/assets/stylesheets/application.css.scss

```diff
@@ -548,11 +550,25 @@ a.btn.btn-default:hover {
 //=========================================== apprentice profiles
 
 
-div#profile-container { display: flex; justify-content: space-between; }
+div#profile-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  @media (min-width: $screen-sm-min) {
+    flex-direction: row;
+  }
+}
 div#profile-container figure {
-  display: inline-block;
+  margin-bottom: 4rem;
+  &:last-child {
+    margin-bottom: 0;
   }
+  @media (min-width: $screen-sm-min) {
+    margin-bottom: 0;
+  }
+}
 div#profile-container figure div {
+  margin: 0 auto;
```

Comment:
Properties should be ordered background-blend-mode, border, border-radius, height, margin, overflow, transition, width